src/exeptions/api.error.js

```diff
@@ -0,0 +1,33 @@
+class ApiError extends Error {
+  constructor({ message, status, errors = {} }) {
+    super(message);
+
+    this.status = status;
+    this.errors = errors;
+  }
+
+  static badRequest(message, errors) {
+    return new ApiError({
+      message,
+      errors,
+      status: 400,
+    });
+  }
+
+  static unauthorized(errors) {
+    return new ApiError({
+      message: 'unauthorized user',
+      errors,
+      status: 404,
+    });
+  }
+  static notFound(errors) {
+    return new ApiError({
+      message: 'not found',
+      errors,
+      status: 401,
```

Comment:
The status code for `notFound` should be 404, not 401. Update `status: 401` to `status: 404`.